### PR TITLE
pacstall: add page

### PR DESCRIPTION
**Pull request by Fabiokleis**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by nicokosi** [comment]
`pacman` seems to a be Linux-only, so I think it that this new page should be ` pages/linux/pacstall.md` (and not `pages/common/pacstall.md`)

**Comment by Fabiokleis** [comment]
@nicokosi ok, I'll move to

File: pages/linux/pacstall.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# pacstall
+
+> Pacstall, making easier to install programs.
```
Comment (blueskyson) [inline pages/linux/pacstall.md:3]: I'm not sure if `pacstall` includes AUR repositories, its repo says it related to AUR.
```suggestion
> Pacstall, make it easier to install AUR packages for Ubuntu.
```

File: pages/linux/pacstall.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -U`
+
+- Update packages, its the same of apt upgrade:
+
+`pacstall -Up`
+
+- Show package info:
+
+`pacstall -Qi {{packagename}}`
```
Comment (blueskyson) [inline pages/linux/pacstall.md:32]: Use snake_case.
```suggestion
`pacstall -Qi {{package_name}}`
```

File: pages/linux/pacstall.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all packages are installed:
```
Comment (blueskyson) [inline pages/linux/pacstall.md:34]:
```suggestion
- List all installed packages:
```

File: pages/linux/pacstall.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# pacstall
+
+> Pacstall, making easier to install programs.
+> More information: <https://github.com/pacstall/pacstall>.
+
+- Search for program in repositories, its equivalent of apt search:
+
+`pacstall -S program`
+
+- Install program, its the same of apt install:
```
Comment (blueskyson) [inline pages/linux/pacstall.md:10]:
```suggestion
- Install program, equivalent of apt install:
```

File: pages/linux/pacstall.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -I program`
+
+- Remove program, its the same of apt remove:
```
Comment (blueskyson) [inline pages/linux/pacstall.md:14]:
```suggestion
- Remove program, equivalent of apt remove:
```

File: pages/linux/pacstall.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Add one more repository to search:
+
+`pacstall -A {{https://github/gitlab.com/example}}`
+
+- Update pacstall's scripts:
+
+`pacstall -U`
+
+- Update packages, its the same of apt upgrade:
```
Comment (blueskyson) [inline pages/linux/pacstall.md:26]:
```suggestion
- Update packages, equivalent of apt upgrade:
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Here are some suggestions.

**Review by blueskyson — APPROVED** [review]
👍

File: pages/linux/pacstall.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# pacstall
+
+> Pacstall, make it easier to install AUR packages for Ubuntu.
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:3]: I think this is a better command description
```suggestion
> An AUR package manager for Ubuntu.
```

File: pages/linux/pacstall.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+> More information: <https://github.com/pacstall/pacstall>.
+
+- Search for program in repositories, its equivalent of apt search:
+
+`pacstall -S program`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:8]: We prefer long arguments if available
```suggestion
`pacstall --search {{package_name}}`
```

File: pages/linux/pacstall.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- Install program, equivalent of apt install:
+
+`pacstall -I program`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:12]: We use token syntax for argument options
```suggestion
`pacstall --install {{package_name}}`
```

File: pages/linux/pacstall.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+- Remove program, equivalent of apt remove:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:14]:
```suggestion
- Remove a package:
```

File: pages/linux/pacstall.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -R program`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:16]:
```suggestion
`pacstall --remove {{package_name}}`
```

File: pages/linux/pacstall.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+- Add one more repository to search:
+
+`pacstall -A {{https://github/gitlab.com/example}}`
+
+- Update pacstall's scripts:
+
+`pacstall -U`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:24]:
```suggestion
`pacstall --update`
```

File: pages/linux/pacstall.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update packages, equivalent of apt upgrade:
+
+`pacstall -Up`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:28]:
```suggestion
`pacstall --upgrade`
```

File: pages/linux/pacstall.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+- Show package info:
+
+`pacstall -Qi {{package_name}}`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:32]:
```suggestion
`pacstall --query-info {{package_name}}`
```

File: pages/linux/pacstall.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all installed packages:
+
+`pacstall -L`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:36]:
```suggestion
`pacstall --list`
```

File: pages/linux/pacstall.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
+# pacstall
+
+> Pacstall, make it easier to install AUR packages for Ubuntu.
+> More information: <https://github.com/pacstall/pacstall>.
+
+- Search for program in repositories, its equivalent of apt search:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:6]:
```suggestion
- Search the package database for a package name:
```

File: pages/linux/pacstall.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -S program`
+
+- Install program, equivalent of apt install:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:10]:
```suggestion
- Install a package:
```

File: pages/linux/pacstall.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -I program`
+
+- Remove program, equivalent of apt remove:
+
+`pacstall -R program`
+
+- Add one more repository to search:
+
+`pacstall -A {{https://github/gitlab.com/example}}`
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:20]: We use this placeholder in the `git-clone` page, I think it fits here.
```suggestion
`pacstall --add-repo {{remote_repository_location}}`
```

File: pages/linux/pacstall.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update pacstall's scripts:
+
+`pacstall -U`
+
+- Update packages, equivalent of apt upgrade:
+
+`pacstall -Up`
+
+- Show package info:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:30]:
```suggestion
- Display information about a package:
```

File: pages/linux/pacstall.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+- Install program, equivalent of apt install:
+
+`pacstall -I program`
+
+- Remove program, equivalent of apt remove:
+
+`pacstall -R program`
+
+- Add one more repository to search:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:18]:
```suggestion
- Add a repository to the database (only GitHub and GitLab are supported):
```

File: pages/linux/pacstall.md [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+
+`pacstall -A {{https://github/gitlab.com/example}}`
+
+- Update pacstall's scripts:
+
+`pacstall -U`
+
+- Update packages, equivalent of apt upgrade:
```
Comment (navarroaxel) [inline pages/linux/pacstall.md:26]:
```suggestion
- Update all packages:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Nice command.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**